quality/lint.xml

```diff
@@ -3,29 +3,7 @@
   <issue id="GoogleAppIndexingWarning" severity="informational" />
   <issue id="AllowBackup" severity="informational" />
   <issue id="UnusedResources">
-    <!-- TODO: Remove these once Material theming migration is finished -->
-    <ignore path="src/main/res/color/contact_doctor_button.xml" />
-    <ignore path="src/main/res/color/material_button_background.xml" />
-    <ignore path="src/main/res/color/selectcountry_radio_button.xml" />
     <ignore path="src/main/res/values/dimens.xml" />
```

Comment:
Hmm, why is this file in unused resources? It is something that is referenced, right?

Reply:
I think it's probably some of the dimens that are being unused. Let me clean that up as well while at it.